kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -415,8 +429,8 @@ define([
             return {
                 content: layout,
                 layout: orderedParams,
-                view: view,
-                paramMap: paramMap,
+                view,
```

Comment:
love the way that `content` comes from the variable `layout`, and `layout` comes from `orderedParams`. 🤯

Reply:
I agree with your 🤯